libs/http-service/src/cli.cpp

```diff
@@ -18,56 +18,103 @@ struct ServeCommand
     int port_ = 0;
     std::vector<std::string> datasourceHosts_;
     std::vector<std::string> datasourceExecutables_;
+    std::string cacheType_;
+    std::string cachePath_;
+    int64_t cacheMaxTiles_ = 1024;
+    bool clearCache_ = false;
     std::string webapp_;
 
-    explicit ServeCommand(CLI::App& app) {
+    explicit ServeCommand(CLI::App& app)
+    {
         auto serveCmd = app.add_subcommand("serve", "Starts the server.");
-        serveCmd->add_option("-p,--port", port_, "Port to start the server on. Default is 0.")->default_val("0");
-        serveCmd->add_option("-d,--datasource-host", datasourceHosts_, "Datasources for the server in format <host:port>. Can be specified multiple times.");
-        serveCmd->add_option("-e,--datasource-exe", datasourceExecutables_, "Datasource executable paths, including arguments, for the server. Can be specified multiple times.");
-        serveCmd->add_option("-w,--webapp", webapp_, "Serve a static web application, in the format [<url-scope>:]<filesystem-path>.");
+        serveCmd->add_option(
+            "-p,--port",
+            port_,
+            "Port to start the server on. Default is 0.")
+            ->default_val("0");
+        serveCmd->add_option(
+            "-d,--datasource-host",
+            datasourceHosts_,
+            "Data sources in format <host:port>. Can be specified multiple times.");
+        serveCmd->add_option(
+            "-e,--datasource-exe",
+            datasourceExecutables_,
+            "Data source executable paths, including arguments. "
+            "Can be specified multiple times.");
+        serveCmd->add_option(
+            "-c,--cache-type", cacheType_, "From [memory|rocksdb], default memory.")
+            ->default_val("memory");
+        // TODO should RocksDB be the default cache?
```

Comment:
I vote yes 👍 But we should document the default somehow (also the default max cache size and path).